client/odysseus/message/style.scss

```diff
@@ -0,0 +1,88 @@
+.chatbox-message {
+	display: flex;
+	justify-content: space-between;
+	align-items: self-start;
+	margin-bottom: 8px;
+	max-width: 70%;
+	word-wrap: break-word;
+	padding: 12px;
+	border-radius: 4px;
+	font-size: $font-body-small;
+	line-height: 1.4;
```

Comment:
We may want to reuse whatever possible from Calypso in terms of variables, specifically for typography and colors.